app.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, render_template, request
+import sys
+
+sys.path.insert(0, 'src')
+
+from contacts_book import *
+from contact import * 
+
+app = Flask(__name__)
+
+contacts_book = ContactsBook()
+
+@app.route('/')
+def index():
+    return render_template('home.html', contacts = contacts_book.get_contacts())
+
+@app.route('/add', methods=['POST'])
```

Comment:
Not too bothered by this, but it might be better to call this route `/contacts`.
I would also replace `/` with `/contacts`.

That way, you have a simply have one endpoint, `/contacts`. You can 'see' all contacts with a `GET` `/contacts` and add a new one with a `POST` `/contacts`

Reply:
Done.